viewer/viewer-awt/src/main/java/org/icepdf/ri/common/SwingViewBuilder.java

```diff
@@ -631,7 +631,7 @@ public JMenuItem buildOpenFileMenuItem() {
 
 
     public JMenuItem buildSendByMailMenuItem() {
-        final JMenuItem mi = makeMenuItem(messageBundle.getString("viewer.menu.sendMail.label"), "sendmail", iconSize,
+        final JMenuItem mi = makeMenuItem(messageBundle.getString("viewer.menu.sendMail.label"), null, iconSize,
```

Comment:
I've made a request to the contributor of this code for the image which I unfortunately missed in the original review.   But null for now works for me.